### DISCUSSION TITLE
ANY FREE FAUCETS ? FOR SEPOLIA ETH ,

### DISCUSSION THREAD

**Discussion by Vishalll07** [Cyfrin/foundry-full-course-cu#1888] [Q&A] [answered]
I am learning web3 but only time I get demotivated is by deploying my smart contract because I am unable to find free faucet. please someone help me with a free faucet

**Comment by EngrPips** [comment, marked as answer]
Hello @Vishalll07, Here `https://faucets.chain.link/sepolia.`

**Reply by Vishalll07** [reply]
hi @EngrPips  thanks you so much

**Reply by EngrPips** [reply]
Anytime, Friend.

**Reply by Vishalll07** [reply]
hey @EngrPips  sorry to bother you one more time but can you tell me why I am not able to view my transactions at EtherScan. 
The Etherscan option isn't available in my transaction console in Remix IDE too.
After your provided solution for sepolia eth .
I changed my network to Arbitrium Sepolia or I guess there was no option for only sepolia so I chose that .
Then it provided me with 25 test links and 0.1 eth.
then I thought it's normal and deployed my contract at remix .
It got deployed and metamask poped up for confirming the transaction.
and I guess it worked too. but after copy pasting my address in EtherScan I am not able to view my transactions .
please guide me if any thing I did wrong here.
![eth1](https://github.com/Cyfrin/foundry-full-course-cu/assets/78494000/fb092f3f-47dc-47c1-841d-49ab9fe57f83)
![eth2](https://github.com/Cyfrin/foundry-full-course-cu/assets/78494000/a81e3ff0-5cf9-43fb-b785-c9e569ba13e2)

**Reply by EngrPips** [reply]
What chain are you running on in `remix` ide?

**Reply by Vishalll07** [reply]
chain ? I am using Injected provider metamask

**Reply by EngrPips** [reply]
Alright, copy the address of the contract you are interacting with and look that up in `etherscan.` let's see if the transactions of the contract would be populated

**Reply by Vishalll07** [reply]
bro , here in the remix the contract transaction is being succeeded but not reflected in etherscan
![image](https://github.com/Cyfrin/foundry-full-course-cu/assets/78494000/71d951bc-c132-4658-84f4-34b95c7d2050)
metamask transactions confirmed-
![eth4](https://github.com/Cyfrin/foundry-full-course-cu/assets/78494000/0c09506d-d618-4413-b648-341edb7083e0)
![eth5](https://github.com/Cyfrin/foundry-full-course-cu/assets/78494000/f9840107-c9b6-4d2f-8474-bb1167ca8126)
![eth6](https://github.com/Cyfrin/foundry-full-course-cu/assets/78494000/25f479e0-518a-436f-b6d5-d23198864047)

**Reply by EngrPips** [reply]
Check on this `https://sepolia.arbiscan.io/`

**Reply by Vishalll07** [reply]
Wow, Thank you so much it worked !!🙂👍
![image](https://github.com/Cyfrin/foundry-full-course-cu/assets/78494000/289d89ca-afd1-4129-aeaa-9ce0fa5d9d5f)
Do you have any social media handles I can follow? at X?

**Reply by EngrPips** [reply]
Yeah, sure, Discord (engr.pips) and X (@EngrPips)
